packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1725,6 +1725,7 @@ export class ContainerRuntime
 				return create404Response(request);
 			}
 			if (this.requestHandler !== undefined) {
+				// eslint-disable-next-line @typescript-eslint/return-await
```

Comment:
Any reason not to update these?

Reply:
I think tests fail when this is changed, but I also thought I left a comment to that effect, so I'll double-check.